types/index.d.ts

```diff
@@ -126,13 +126,7 @@ interface BaseOptions<T extends PostgresTypeList> {
   keep_alive: number | null;
 }
 
-interface PostgresTypeList {
-  [name: string]: postgres.PostgresType;
-}
-
-interface JSToPostgresTypeMap {
-  [name: string]: unknown;
-}
+interface PostgresTypeList extends Record<string, postgres.PostgresType> {}
```

Comment:
I have updated the `PostgresTypeList` to extend a `Record`.

Reply:
The issue with `PostgresTypeList` is that it's confusing for many users, so the best would be to just replace this type with `Record<string, postgres.PostgresType>`, just like you did with `JSToPostgresTypeMap` :+1: